app-httpd.conf

```diff
@@ -0,0 +1,28 @@
+<VirtualHost *:80>
+    LoadModule proxy_module modules/mod_proxy.so
+    LoadModule proxy_http_module modules/mod_proxy_http.so
+    LoadModule proxy_wstunnel_module modules/mod_proxy_wstunnel.so
+    LoadModule rewrite_module modules/mod_rewrite.so
+    LoadModule include_module modules/mod_include.so
+
+    DocumentRoot /usr/local/apache2/htdocs/gridapp/
+    AddOutputFilterByType INCLUDES text/html
+    <Directory /usr/local/apache2/htdocs/gridapp/>
+        Options +Includes
+    </Directory>
+
+    RewriteEngine On
+    RewriteMap remapbase "prg:/bin/sed -u -e 's;[^ ]* ;;' -e 's; .*;;' -e 's;[^/]*;..;g' -e 's;../..;;' -e 's/.//' -e 's;^$;.;'"
+
+    RewriteCond /usr/local/apache2/htdocs/gridapp%{REQUEST_URI} -f [OR]
+    RewriteCond %{REQUEST_URI} ^/api/.* [OR]
+    RewriteCond %{REQUEST_URI} ^/ws/.*
+    RewriteRule ^ - [S=2]
+    RewriteRule ^ "${remapbase:%{THE_REQUEST}}" [DPI]
+    RewriteRule "^(.*)$" /index.html [E=BASE:$1,L]
+    SetEnvIf BASE ^$ BASE=.
+
+    ProxyRequests off
+    ProxyPassMatch "^/api/(case-server|study-server|gateway)/(.*)$" http://$1/$2
```

Comment:
@geofjamg I haven't changed it because I'm not sure there is  **merge-server | getway**

Reply:
it is merge-orchestrator-server